app/serializers/cache/heading_serializer.rb

```diff
@@ -119,12 +119,27 @@ def as_json
                                 [
                                   { measure_type: [:measure_type_description] },
                                   { measure_components: [{ duty_expression_description: :duty_expression_description }, :measurement_unit_qualifier] },
+                                  { additional_code: [:additional_code_description] },
                                 ],
                               )
                             else
                               commodity.overview_measures
                             end
         commodity_attributes[:overview_measures] = overview_measures.map do |measure|
+          additional_code = if measure.additional_code
+                              {
+                                additional_code_id: measure.additional_code_sid.to_s,
```

Comment:
JSONAPI ids automatically get to_s called on them. You can remove the to_s here and on line 133